pandas/tests/indexes/common.py

```diff
@@ -886,7 +886,7 @@ def test_hasnans_isnans(self):
                 assert not idx.hasnans
 
                 idx = index.copy()
-                values = idx.values
+                values = np.asarray(idx.values)
```

Comment:
Made this change since `IntervalArray` currently isn't mutable. Worth talking about that.